src/v/cloud_io/remote.cc

```diff
@@ -67,6 +68,7 @@ cloud_io::provider infer_provider(
     switch (backend) {
     case model::cloud_storage_backend::aws:
```

Comment:
wrong one here

Reply:
Oh wow. Fixed but testing locally to make sure this would have failed tests.